api/AnimationEffectTimingReadOnly.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "AnimationEffectTimingReadOnly": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEffectTiming",
```

Comment:
This should link to https://developer.mozilla.org/en-US/docs/Web/API/AnimationEffectTimingReadOnly